docs/API_ACCESS.md

```diff
@@ -0,0 +1,62 @@
+The Search API is implemented with GraphQL. This document explains how to access the service to send requests.
+
+## Pre-requisite
+Red Hat Advanced Cluster Management installed on your OpenShift cluster.
+
+## Locate the Search API endpoint
+### From within the cluster
+If your application is deployed in the cluster, and it has the correct permissions, it can send requests to https://search-search-api.open-cluster-management.svc:4010/searchapi/graphql
+
+### From outside the cluster
+By default, the Search API is not exposed. Follow [this topic](https://docs.openshift.com/container-platform/4.9/networking/routes/secured-routes.html) to create a secure route to the service `search-search-api` in the `open-cluster-management` namespace.
+
+```
+# NOTE: this sample requires an insecure connection from the client.
```

Comment:
don't follow why this requires an insecure connection from client. Its fully SSL traffic here.

Reply:
This example needs insecure connection because the certificates generated by the Service Serving certificate are only valid for the internal DNS name and only intended for internal communication. [Documentation link](https://docs.openshift.com/container-platform/4.7/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate)

The alternative is to provide the certificate when creating the route.  I included the link to the documentation for creating a secure route in the statement above.

I'm using insecure connection in this sample to keep it simple, but adding the reminders that a secure connection must be used in production.